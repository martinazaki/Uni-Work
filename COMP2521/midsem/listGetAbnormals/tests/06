9 13 8 5 1 6 9 13 7
4
