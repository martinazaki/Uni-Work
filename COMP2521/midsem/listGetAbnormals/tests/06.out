> Enter list: > Enter threshold: 
-----------------
>> Original list:
Size: 9
Forwards:  [9] -> [13] -> [8] -> [5] -> [1] -> [6] -> [9] -> [13] -> [7] -> X
Backwards: [7] -> [13] -> [9] -> [6] -> [1] -> [5] -> [8] -> [13] -> [9] -> X

>> Threshold: 4
-----------------
>> Abnormals:
Size: 3
Forwards:  [13] -> [1] -> [13] -> X
Backwards: [13] -> [1] -> [13] -> X
